src/action/payment.js

```diff
@@ -165,14 +165,35 @@ class PaymentAction {
       return this._notification.display({ msg: 'Enter an invoice or address' });
     }
     if (await this.decodeInvoice({ invoice: this._store.payment.address })) {
-      this._nav.goPayLightningConfirm();
+      if (
+        this._store.payment.amount === '0' ||
+        this._store.payment.amount === 0
```

Comment:
is there a need for checking for both `0` and `'0'`? I feel like it should always just be a value of one type. If that's not the case then it should be cast to one type first before checking.

Reply:
Not really, I should :tm: be '0' its just me being defensive. Will fix.

Reply:
Fixed in d1c9dfe8